src/cmd/src/options.rs

```diff
@@ -36,6 +37,18 @@ pub struct MixOptions {
     pub logging: LoggingOptions,
 }
 
+impl From<MixOptions> for FrontendOptions {
+    fn from(value: MixOptions) -> Self {
+        value.frontend
+    }
+}
+
+impl TomlSerializable for MixOptions {
+    fn to_toml(&self) -> String {
+        toml::to_string(self).unwrap()
```

Comment:
Better to return an error or a default string with the error. As we don't want to panic here.
```rust
match toml::to_string() {
    Ok(s) => s,
    Err(e) => format!("Failed to serialize options to TOML: {e}"),
}
```